scipy/stats/_qmc.py

```diff
@@ -172,7 +172,7 @@ def scale(
 
     if not reverse:
         # Checking that sample is within the hypercube
-        if not (np.all(sample >= 0) and np.all(sample <= 1)):
+        if not ((sample.max() <= 1.) and (sample.min() >= 0.)):
```

Comment:
Do we need to make `_qmc` changes in the same PR as the spatial changes? Can they stand on their own justification?

Reply:
I can remove the change if you want, it's here because we talked about this enhancement when the new function was still in QMC. I just did not want to make a PR just for that.

Reply:
@tupui if you make this as a separate PR, I can merge it right away and we can close this comment.

Reply:
Ok then I will make another PR soon.